gateway/mw_context_vars.go

```diff
@@ -18,7 +18,7 @@ func (m *MiddlewareContextVars) Name() string {
 }
 
 func (m *MiddlewareContextVars) EnabledForSpec() bool {
-	return m.Spec.EnableContextVars
+	return true
```

Comment:
Why does `MiddlewareContextVars.EnabledForSpec` return always `true`?

Reply:
The idea for streaming is to have context vars always enabled. There won't be an option to disable it in UI